src/nih_wayfinding/app/scripts/navigation/navigation-service-slopedmock.js

```diff
@@ -0,0 +1,201 @@
+
+(function () {
```

Comment:
If we're going to check this in, add some detailed notes in a header comment about why it's here, what it's for and the conditions under which it could be removed.